5/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mhart/alpine-node:5
+
+RUN apk update
+
+# npm install deps
+RUN apk add git openssh-client
+
+# native build deps
+RUN apk add make gcc
+
+# scripting tools
+RUN apk add bash
+
+# update NPM
+RUN npm install -g npm@3.0-latest
```

Comment:
Is there a 3-latest we can use, so we get minor version updates? Not sure what they publish. If not, disregard :)